doc/en/user/source/rest/api/stylegenerator.rst

```diff
@@ -0,0 +1,125 @@
+.. _rest_api_stylegenerator:
+
+Style Generator
+===============
+
+The Style Generator endpoint is used for generating default styles. Styles generated using this endpoint may be added to geoserver using the :ref:`styles <rest_api_styles>` endpoint.
+
+``/stylegenerator[.<format>]``
+------------------------------
```

Comment:
Interesting, I may of done this as an endpoint on the styles REST API endpoint.

Something like PUT /workspaces/<ws>/styles/<s>[.<format>] would complete the table?